src/tablePrinter.ts

```diff
@@ -23,11 +23,28 @@ const getColumnsCanFitInPage = (
   // get page width
   const availablePageWidth = getPageAvailableWidth(doc, table)
   let remainingWidth = availablePageWidth
+  // get column data key to repeat
+  const horizontalPageBreakRepeat = table.settings.horizontalPageBreakRepeat;
+  let repeatColumnIndex = -1;
```

Comment:
I think we can simplify some code here if we remove the repeatColumnIndex in favour of moving `repeatCol` outside of the if statement. Something like this:

```
const repeatColumn = null
if (horizontalPageBreakRepeat) {
    repeatColumn = table.columns.find((item) => item.dataKey == horizontalPageBreakRepeat);
    // ...
  }
  while (i < len) {
    if (repeatColumn?.index === i) {
    // ...
```